src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -614,11 +537,18 @@ private InstructorFeedbackResultsGroupByParticipantPanel buildGroupByParticipant
                                                         response.feedbackSessionName,
                                                         Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS);
             
+            int sectionId = 0;
+            Pattern pattern = Pattern.compile("^section-(\\d+)");
```

Comment:
Let's move this to be a static field, we don't have to initialise this each time